cachelib/adaptor/rocks_secondary_cache/README.md

```diff
@@ -0,0 +1,44 @@
+# RocksDB CacheLib Secondary Cache
+
+This directory contains files and tests for building Cachelib as a SecondaryCache for RocksDB.
+
+# Build
+
+Currently the build is only supported via cmake.
+
+First, build cachelib through the normal build procedure.
+
+To build under RocksDB, link this directory adaptor/rocksdb_secondary_cache) to the plugins/cachelib directory under RocksDB:
+```
+$ln -s .../secondary_cache .../plugins/cachelib
+```
+This will allow RocksDB to find and build the CacheLib plugin code.
+
+Next, under the RocksDB build directory, instruct RocksDB to build and include the cachelib plugin:
```

Comment:
I'm curious, how is the folly dependency handled? Should the user do this first - https://github.com/facebook/rocksdb/wiki/RocksDB-Contribution-Guide#build-rocksdb-with-folly-integration-enabled?